cmd/osbuild-composer/main.go

```diff
@@ -72,7 +72,7 @@ func main() {
 	// Optionally run RCM API as well as Weldr API
 	if len(listeners) == 3 {
 		rcmListener := listeners[2]
-		rcmAPI := rcm.New(logger, store)
+		rcmAPI := rcm.New(logger, store, rpmmd.NewRPMMD())
```

Comment:
`rpm` variable can be reused here instead of creating new `RPMMD` instance

Reply:
yeah, I was thinking about it. Is it safe to share it?

Reply:
`type rpmmdImpl struct{}` - it doesn't have much state. :-D